dev-libs/blazesym_c/blazesym_c-0.1.1.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Autogenerated by pycargoebuild 0.14.0
+
+EAPI=8
+
+CRATES="
+"
+
+declare -A GIT_CRATES=(
+	[criterion-plot]='https://github.com/bheisler/criterion.rs;b913e232edd98780961ecfbae836ec77ede49259;criterion.rs-%commit%/plot'
+	[criterion]='https://github.com/bheisler/criterion.rs;b913e232edd98780961ecfbae836ec77ede49259;criterion.rs-%commit%'
+	[vmlinux]='https://github.com/libbpf/vmlinux.h;a9c092aa771310bf8b00b5018f7d40a1fdb6ec82;vmlinux.h-%commit%'
+)
+
+inherit cargo rust-toolchain
+
+DESCRIPTION="C bindings for blazesym, a library for address symbolization and related tasks"
+HOMEPAGE="https://github.com/libbpf/blazesym"
+SRC_URI="
+	https://github.com/libbpf/blazesym/archive/refs/tags/capi-v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/gentoo-crate-dist/blazesym/releases/download/capi-v${PV}/blazesym-capi-v${PV}-crates.tar.xz
+	${CARGO_CRATE_URIS}
+"
+if [[ ${PKGBUMPING} != ${PVR} ]]; then
+	SRC_URI+="
+		blazesym-capi-v0.1.1-crates.tar.xz
+	"
+fi
```

Comment:
Do you really need it now, since we use pre-prepared crates tarball

Reply:
I have no idea, that got added by pycargoebuild. I guess I can remove it, yes.